.github/workflows/documentation-windows.yml

```
@@ -2,7 +2,7 @@ name: Documentation Windows
 
 on:
   push:
-    branches: [ master, develop, debug_windows_doc ]
+    branches: [ master, develop, 9818_Windows_documentation ]
```

Before merging, apply this:

```suggestion
    branches: [ master, develop ]
```